schemas/mss.xml

```diff
@@ -541,11 +638,18 @@ xml:lang="en">
                     </attList>
                 </elementSpec>
 
+<!--Notes on presence of palimpsest to "condition"-->
+
                 <!--          controlled values for condition-->
                 <elementSpec ident="condition" mode="change" module="msdescription">
                     <attList>
                         <attDef ident="key" mode="replace" usage="req">
                             <valList type="closed">
+                                
+                                <!--Additional values:
+                                -	Part of codex
+                                -	Fragment of a leaf/sheet
+                                -->
```

Comment:
Great idea. We could simply add `@scope` to `extent` using the values you suggested.

Reply:
Actually I overlooked the fact that `<damage>` can be used inside `<condition>` (or inside `<foliation>` for that matter, I'm not sure which is better for us). E.g., `<damage unit="leaf" quantity="5" type="missing"/>` with `@quantity` optional in case it can be determined. Also `<locus>` can be used inside damage to indicate where the missing portion is.